lib/cb/utils/validator.rb

```diff
@@ -65,7 +65,13 @@ def try_parse_xml(body)
       end
 
       def error_message(processed_response)
-        processed_response.fetch('errors', processed_response.fetch('Errors', ''))
+        return '' if processed_response.empty?
+        find_errors_node(processed_response) || ''
+      end
+
+      def find_errors_node(processed_response)
+        nested_hash = processed_response[processed_response.keys.first]
```

Comment:
if we wanna update those tests to have more than one key in the response then this would be the function:

```
      def error_message(processed_response)
        return '' if processed_response.empty?
        if processed_response.keys.count > 1
          processed_response['errors'] || processed_response['Errors'] || ''
        else
          nested_hash = processed_response[processed_response.keys.first]
          nested_hash['errors'] || nested_hash['Errors'] || ''
        end
      end
```